### PR TITLE
CSS namespace

### PR DESCRIPTION
**Pull request by Nazeh**
**Link to issue:** https://github.com/Doist/reactist/issues/174

**Short description:**

 Changed namespacing from `.reactist.popover {}` to `.reactist_popover {}`.

**PR Checklist:**

-   [ ] Added tests for bugs / new features
-   [x] Updated docs (storybooks, readme)
-   [x] Described changes in CHANGELOG.md
-   [x] Executed `npm run check` and made sure no errors/warnings were shown
-   [x] Executed `npm run test` and made sure all tests are passing
-   [x] Bumped version in package.json
-   [x] Updated all static build artifacts (`npm run build-all`)

### CODE REVIEW

**Comment by stkao05** [comment]
Thanks for your contribution. Looks good to me. Could you rebase and squash the commits?

**Comment by Nazeh** [comment]
@stkao05 Thank you for the quick review. Please check out if it is ready now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Reactist follows [semantic versioning](https://semver.org/) and doesn't introduce breaking changes (API-wise) in minor or patch releases. However, the appearance of a component might change in a minor or patch release so keep an eye on redesigns and make sure your app still looks and feels like you expect it.
 
+## 2.0.3
+
+- [Tweak] Changed namespacing from `.reactist.popover {}` to `.reactist_popover {}`.
+
```
Comment (gnapse) [inline CHANGELOG.md]: This is not a minor change. This is actually a breaking change. All apps using reactist and customizing some styles will need to update their customizations.

Comment (Nazeh) [reply CHANGELOG.md]: Updated the version to 3.0.0 and waiting for @henningmu's review.

**Comment by henningmu** [comment]
Hi @Nazeh,

I'm back from my vacation and took note of this PR. As it's quite the large change and might change some of our internal usages I want to thoroughly review it. Except some movement here until end of week latest 👍

**Comment by Nazeh** [comment]
Welcome back @henningmu,
Looking forward to it.

**Pull request closed without merging**